File: android/app/src/main/java/com/example/fugliara/ccs002/server/dataobjects/UserRatingItem.java
package com.example.fugliara.ccs002.server.dataobjects;

public class UserRatingItem {
    @com.google.gson.annotations.SerializedName("id")
    public String id;
    @com.google.gson.annotations.SerializedName("comments")
    public String comments;
    @com.google.gson.annotations.SerializedName("value")
    public int value;
    @com.google.gson.annotations.SerializedName("userSource")
    public UserItem userSource;
    @com.google.gson.annotations.SerializedName("userDestin")
    public UserItem userDestin;
    @com.google.gson.annotations.SerializedName("orderItem")
    public OrderItem orderItem;
    @com.google.gson.annotations.SerializedName("dateTime")
    public String dateTime;

    public UserRatingItem() {
        id = "";
        comments = "";
        value = 0;
        userDestin = null;
        userSource = null;
        orderItem = null;
        dateTime = "";
    }
}

File: azure/transporterSolution/transporterService/DataObjects/UserRatingItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using transporterService.DataTranspObjects;

namespace transporterService.DataObjects
{
    public partial class UserRatingItem : BaseItem
    {
        public string Comments { get; set; }
        public int Value { get; set; }
        public virtual UserItem UserSource { get; set; }
        public virtual UserItem UserDestin { get; set; }
        public virtual OrderItem OrderItem { get; set; }
        public string DateTime { get; set; }
    }
}